src/Game/UI/Gumps/CharCreation/CreateCharTradeGump.cs

```diff
@@ -250,7 +251,13 @@ private bool ValidateValues()
             }
             else
             {
-                UIManager.GetGump<CharCreationGump>()?.ShowMessage(ClilocLoader.Instance.GetString(1080032));
+                UIManager.GetGump<CharCreationGump>()
+                         ?.ShowMessage
+                         (
+                             Client.Version <= ClientVersion.CV_5090 ?
+                                 "You must have three unique skills chosen!" :
```

Comment:
ClassicUO now supports Resources files to manage hardcoded strings like above.
Can you add it to ResGump res file?

Reply:
Done!